argilla-frontend/README.md

```diff
@@ -31,8 +31,7 @@
 </a>
 </p>
 
-Argilla is a **collaboration platform for AI engineers and domain experts** that require **high-quality outputs, full data ownership, and overall efficiency**.
-
+Argilla is a collaboration tool for AI engineers and domain experts who need to build high-quality datasets for their projects.
 If you just want to get started, we recommend our [UI demo](https://demo.argilla.io/sign-in?auth=ZGVtbzoxMjM0NTY3OA%3D%3D) or our [free Hugging Face Spaces deployment integration](https://huggingface.co/new-space?template=argilla/argilla-template-space). Curious, and want to know more? Read our [documentation](https://argilla-io.github.io/argilla/latest/).
```

Comment:
@davidberenstein1957 @frascuchon Will these auth links work with the changes to `demo.argilla.io`?

Reply:
@burtenshaw I'd like to replace this link [free Hugging Face Spaces deployment integration](https://huggingface.co/new-space?template=argilla/argilla-template-space).

with the docs.argilla.io/latest/getting_started/quickstart of the docs. the reasoning is that without context people tends to get confused when they see the space creation wizard

Reply:
@dvsrepo I understand that people would get confused by the config. Let's change `https://huggingface.co/new-space?template=argilla/argilla-template-space`.

Reply:
> @davidberenstein1957 @frascuchon Will these auth links work with the changes to demo.argilla.io?

The Cloudflare redirect will respect the path and query string so I guess it should work.